build/Makefile

```diff
@@ -464,14 +464,13 @@ setup-grafana: PVC ?= true
 setup-grafana: PV_SIZE ?= 64Gi
 setup-grafana: PASSWORD ?= admin
 setup-grafana:
-	-$(DOCKER_RUN) kubectl create namespace metrics
-	$(DOCKER_RUN) helm repo add stable https://kubernetes-charts.storage.googleapis.com/
+	$(DOCKER_RUN) helm repo add grafana https://grafana.github.io/helm-charts
 	$(DOCKER_RUN) helm repo update
 	$(DOCKER_RUN) kubectl apply -f $(mount_path)/build/grafana/
-	$(DOCKER_RUN) helm upgrade grafana stable/grafana \
-		--install --wait --namespace metrics \
-		--set persistence.enabled=$(PVC),server.persistentVolume.size=$(PV_SIZE) \
-		--set adminPassword=$(PASSWORD) -f $(mount_path)/build/grafana.yaml
+	$(DOCKER_RUN) helm upgrade --install --wait grafana grafana/grafana --version=5.7.10 \
+					--namespace metrics --create-namespace \
```

Comment:
The indentation here seems off (and not consistent with how it's done for the prometheus command above).

Reply:
Fixed!